core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/stack/cluster/gateway/GatewayJson.java

```diff
@@ -17,7 +17,7 @@
 import io.swagger.annotations.ApiModelProperty;
 
 @ApiModel
-@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonIgnoreProperties(ignoreUnknown = true, value = "path", allowSetters = true)
```

Comment:
ssoProvider is missing

Reply:
I think it is much cleaner if we are putting the JsonIgnore annotation directly on member variable